mods/beds/functions.lua

```diff
@@ -97,13 +99,13 @@ local function update_formspecs(finished)
 
 	if finished then
 		form_n = beds.formspec ..
-			"label[2.7,11; Good morning.]"
+			S("label[2.7,11; Good morning.]")
```

Comment:
this should always be done via code like `"label[2.7,11; " .. S("Good morning.") .. "]"` to make changes of the label properties easy.